config/govuk_index/migrated_formats.yaml

```diff
@@ -420,6 +420,7 @@ non_indexable:
 - social_media_use
 - speech
 - staff_update
+- standard
```

Comment:
This is in the `non_indexable` section.  Is that intentional?

Reply:
Originally that was not intentional but actually, on further inspection the other publication types don't seem to be indexed so we'll go with my original approach. 😄